core/EventDispatcher.php

```diff
@@ -60,7 +60,7 @@ public static function getInstance()
 
     private $pluginHooks = array();
 
-    public static $_SKIP_EVENTS_IN_TESTS = false;
+    public static $skipEventsInTests = false;
```

Comment:
This property could in theory also be used by plugins in their tests. I'm not aware of a plugin, that is doing it, but we shouldn't change that silently in a patch release.

Reply:
Reverted the underscore rule.